web/packages/shared/services/loc/loc.ts

```diff
@@ -83,3 +83,19 @@ export function displayDateTime(date: Date) {
 export function dateToUtc(date: Date) {
   return new Date(date.getTime() + date.getTimezoneOffset() * 60 * 1000);
 }
+
+export function dateTimeShortFormat(date: Date) {
+  try {
+    if (isTest) {
+      return new Intl.DateTimeFormat('en-US', { timeStyle: 'short' }).format(
+        date
+      );
+    }
+    return new Intl.DateTimeFormat(undefined, { timeStyle: 'short' }).format(
+      date
+    );
+  } catch (err) {
+    logger.error('dateTimeShortFormat()', err);
+    return 'undefined';
```

Comment:
1) What errors do we expect to be thrown here? I see that other functions in this file do use `try … catch`, but they also use `format` from `date-fns` and not `Intl.DateTimeFormat`.
2) In case of an error, wouldn't it be better to use some kind of method on `Date` to display something approximate instead of saying `"undefined"`? I'm not sure if it even makes sense, but I assume here that `Intl.DateTimeFormat().format` cannot fail (see question 1). This might not be true though.

Reply:
1. From what I see, the function throws only when `locales` or `options` contain invalid values https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat#exceptions.
If so, should we catch the error at all? It can come only from our invalid configuration, not from the user input.